.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Unit Tests
+
+on: [pull_request]
+
+jobs:
+  test:
+    name: 'Node ${{ matrix.node }}'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14, 16]
```

Comment:
this is fine, but you may want to use my "matrix" action that will auto-update as new node versions are released.

Here's an example of it being used: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/.github/workflows/node-4%2B.yml#L5-L27